agent/consul/state/config_entry_events_test.go

```diff
@@ -547,3 +547,348 @@ func TestServiceDefaultsSnapshot(t *testing.T) {
 		})
 	}
 }
+
+func TestAPIGatewaySnapshot(t *testing.T) {
+	const index uint64 = 123
+
+	ixn1 := &structs.APIGatewayConfigEntry{
```

Comment:
Could this be refactored to maybe `gw1` for the var name (or something similar).

When looking a little further down I thought that maybe we were reusing an intention. Upon closer inspection I suspect this was copy/pasted from an intention test and its just some var names that were untouched.

The same thing goes for the test variables in the other new tests within this file.